source/install.txt

```diff
@@ -80,35 +80,14 @@ Select the appropriate tab for your operating system:
       Install from ``.zip`` File
       ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-      To manually install ``mongosh`` using a downloaded ``.zip``
+      To manually install :binary:`mongosh` using a downloaded ``.zip``
       file:
 
       .. include:: /includes/steps/install-shell-macos.rst
 
    .. tab::
       :tabid: linux
 
-      Considerations for Ubuntu
-      ~~~~~~~~~~~~~~~~~~~~~~~~~
-
-      - ``mongosh`` is available as a
-        :abbr:`PPA (Personal Package Archive)` for Ubuntu 20.04 (Focal)
-        and Ubuntu 18.04 (Bionic).
-        
-        - If you are using Ubuntu 20.04 (Focal) or Ubuntu 18.04
-          (Bionic), use the procedure in the ``.deb`` tab.
-        
-        - If you are using Ubuntu 16.04 (Xenial), you must
-          obtain the ``mongosh`` tarball using the procedure in the
-          ``.tgz`` tab.
-      
-      - The ``mongocryptd`` binary in the |mdb-shell| ``.deb`` 
-        package does not work on Ubuntu 16.04 or older. On 
-        Ubuntu 16.04 and older systems, you must 
-        :manual:`install </reference/security-client-side-encryption-appendix/#installation>` 
-        the ``mongocryptd`` binary separately to use its 
-        functionality in the |mdb-shell|.
-
       Procedure
```

Comment:
It may still be valuable to include a section immediately preceding "Procedure" to list out all of the supported Linux architectures. This way, users can tell at a glance whether their architecture is supported.

Under the `.deb` tab, we mention which platforms users can install on. We say:

> :binary:`mongosh` is available as a :abbr:`PPA (Personal
   Package Archive)` for Ubuntu 20.04 (Focal) and Ubuntu 18.04 (Bionic).

It might make sense to pull that info up higher on the page and create a dedicated section that says "here are the platforms on which you can run monosh".

Something like:

```
Supported Linux Platforms
~~~~~~~~~~~~~~~~~~~~

You can run mongosh on...

- Ubuntu...
- RHEL...
- Amazon...
```

Reply:
The PPA info is needed here because we need to let users know that mongosh isn't in the Ubuntu repo. 

I'll open another ticket for a more general, supported platform list.

Reply:
https://jira.mongodb.org/browse/DOCSP-21075